tofhir-server/src/main/scala/io/tofhir/server/util/CsvUtil.scala

```diff
@@ -206,4 +216,11 @@ object CsvUtil {
 
   }
 
+  /**
+   * Represents a CSV header with a new name and a previously saved name.
+   * @param field The current name of the header.
+   * @param savedName The previously saved name of the header, used for column name change without data loss
+   * If a new header is desired to be created, saved name may be any placeholder
+   */
+  case class CsvHeader(field: String, savedName: String)
```

Comment:
We can simply rename the fields to currentName and previousName to make it clear.

Further, create a separate class for this model under model.csv package.